### PR TITLE
Fix part of #736: Lowfi profile progress

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/7fb5929e-4652-4412-bf8c-64d94f4159f9/UP-User-Profile-

For dummy progress paste this code in HomeFragmentPresenter: https://gist.github.com/rt4914/5258375c8d2cca81b6a5b62ccab8e143

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragmentPresenter.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+/** The presenter for [StoryFragment]. */
+class CompletedStoryListFragmentPresenter @Inject constructor(
+  private val activity: AppCompatActivity,
+  private val fragment: Fragment
+) {
+
+  private lateinit var binding: CompletedStoryListFragmentBinding
+
+  fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?, internalProfileId: Int): View? {
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/completedstorylist/CompletedStoryListFragmentPresenter.kt:16]: The `internalProfileId` is not getting used currently but will be used in next PRs.

File: app/src/main/java/org/oppia/app/profileprogress/ProfileProgressHeaderViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package org.oppia.app.profileprogress
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.ObservableField
+import org.oppia.app.home.RouteToRecentlyPlayedListener
+import org.oppia.app.model.Profile
+
+/** Header [ViewModel] for the recycler view in [ProfileProgressFragment]. */
+class ProfileProgressHeaderViewModel(activity: AppCompatActivity) : ProfileProgressItemViewModel() {
+  // TODO(#777): Edit image and shown full screen image popup.
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/profileprogress/ProfileProgressHeaderViewModel.kt]: Please check this todo. It is not clear.

File: app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+  private fun processOngoingStoryResult(ongoingStoryListResult: AsyncResult<OngoingStoryList>): OngoingStoryList {
+    if (ongoingStoryListResult.isFailure()) {
+      logger.e(
+        "ProfileProgressFragment",
+        "Failed to retrieve ongoing story list: ",
+        ongoingStoryListResult.getErrorOrNull()!!
+      )
+    }
+
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt]: Remove this extra line.

File: app/src/main/res/layout/completed_story_list_fragment.xml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/white">
+
+    <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/topic_app_bar_layout"
```
Comment (veena14cs) [inline app/src/main/res/layout/completed_story_list_fragment.xml]: Change this id to file specific id.

**Review by veena14cs — APPROVED** [review]
LGTM please address some nit changes.

File: app/src/main/java/org/oppia/app/profileprogress/ProfileProgressHeaderViewModel.kt (hunk 2 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/profileprogress/ProfileProgressHeaderViewModel.kt]: `// TODO(#777): Click on ImageView to show a popup which can either edit the image or view image in full screen.`

File: app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt (hunk 3 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/profileprogress/ProfileProgressViewModel.kt]: Removed.

File: app/src/main/res/layout/completed_story_list_fragment.xml (hunk 4 above)
Comment (rt4914) [reply app/src/main/res/layout/completed_story_list_fragment.xml]: Updated to `completed_story_list_app_bar_layout`

**Comment by rt4914** [comment]
Merging this now as it will unlock 5 issues. @nikitamarysolomanpvt You can review this tomorrow and if there are any changes I can update that.

**Review by nikitamarysolomanpvt — COMMENTED** [review]
LGTM

**Pull request merged**